python/ray/autoscaler/sdk.py

```diff
@@ -183,18 +183,24 @@ def get_worker_node_ips(cluster_config: Union[dict, str]) -> List[str]:
 
 def request_resources(num_cpus: Optional[int] = None,
                       bundles: Optional[List[dict]] = None) -> None:
-    """Remotely request some CPU or GPU resources from the autoscaler.
+    """Command the autoscaler to scale to accommodate the specified requests.
 
-    This function is to be called e.g. on a node before submitting a bunch of
-    ray.remote calls to ensure that resources rapidly become available.
+    The cluster will immediately attempt to scale to accommodate the requested
+    resources, bypassing normal upscaling speed constraints. This does not
+    take into account existing resource usage (i.e., the target cluster size is
```

Comment:
The sentence after "i.e." is still confusing. It somehow takes into account the existing resource usage to determine how much to add. (Target = requests, demands = requests - current usage)
I would instead give an example there instead, one with cpus that you have in the code and the other with the bundles from the issue filed recently.